code/edge/src/main/java/com/adobe/marketing/mobile/EdgeExtension.java

```diff
@@ -363,29 +227,28 @@ void handleSharedStateUpdate(final Event event) {
 	 *
 	 * @param event an event containing {@link ExperienceEvent} data for processing; the event and the event data should not be null, checking in listener
 	 */
-	void handleExperienceEventRequest(final Event event) {
+	void handleExperienceEventRequest(@NonNull final Event event) {
 		if (shouldIgnore(event)) {
 			return;
 		}
-
-		processAddEvent(event);
+		processEdgeEvent(event);
 	}
 
 	/**
 	 * Handles the Consent Update event
 	 *
 	 * @param event current event to process; the event and the event data should not be null, checking in listener
 	 */
-	void handleConsentUpdate(final Event event) {
-		processAddEvent(event);
+	void handleConsentUpdate(@NonNull final Event event) {
+		processEdgeEvent(event);
```

Comment:
a bit confusing cause this is not an edge event - might be good to rename it to processEvent

Reply:
These are events which go to the Edge Network. I want to differentiate the "events" here from all the other events processed in this class, so I'd like something less generic than processEvent. What about `processEventToHitQueue` or `sendEventToHitQueue` or `queueEdgeDataEvent` ?

Reply:
or `processAndQueueEvent`